api/src/indigo_molecule.cpp

```diff
@@ -2665,11 +2665,11 @@ CEXPORT int indigoAddSuperatom(int molecule, int natoms, int* atoms,
 int idx = mol.sgroups.addSGroup(SGroup::SG_TYPE_SUP);
 Superatom& satom = (Superatom&)mol.sgroups.getSGroup(idx);
 satom.subscript.appendString(name, true);
-if (atoms == 0)
+if (atoms == nullptr)
     throw IndigoError("indigoAddSuperatom(): atoms were not specified");
 
-for (int i = 0; i < natoms; i++)
-    satom.atoms.push(atoms[i]);
+else
```

Comment:
This `else` is redundant, since other branch of condition leads to exception throwing.

Reply:
This is here by two reasons:
1) Replacing `for` it forms in-line comparison of the next line:
```
   satom.atoms.**push**(atoms[i]);	   <----->         satom.atoms.**concat**(atoms, natoms);
```
2) In case the exception is avoided in the future, it is still forms an else clause.